docs/application/dotnet/guides/applications/service_application.md

```diff
@@ -0,0 +1,213 @@
+
+```csharp
+        protected override void OnLocaleChanged(LocaleChangedEventArgs e)
+        {
+            base.OnLocaleChanged(e);
+        }
+
+        protected override void OnRegionFormatChanged(RegionFormatChangedEventArgs e)
+        {
+            base.OnRegionFormatChanged(e);
+        }
+```
+
+```csharp
+        static void Main(string[] args)
+        {
+            App app = new App();
+            app.Run(args);
+        }
+    }
+}
+```
+
+## Application attributes
+Describe your service application attributes in the manifest file. The attributes determine the application behavior. The following code example illustrates how you can define the attributes:
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns="http://tizen.org/ns/packages" api-version="6" package="org.tizen.example.servicesample" version="1.0.0">
+  <profile name="common" />
+  <service-application appid="org.tizen.example.servicesample"
+					exec="servicesample.dll"
+					type="dotnet"
+					multiple="false"
+					taskmanage="false"
+					nodisplay="true"
+                    api-version="8">
+    <label>servicesample</label>
+    <icon>servicesample.png</icon>
+  </service-application>
+</manifest>
+```
+
+- `auto-restart`
+
+  If set to `true`, the application restarts whenever it terminates abnormally. If the application is running, it is launched after installing or updating the package.
+
+- `on-boot`
+
+  If set to `true`, the application launches on boot time, and after installing or updating the package. The application does not start, if this attribute is removed after updating the package.
+
+The following table defines the behaviors resulting from the attribute combinations:
+
+**Table: Attribute combinations**
+
+| `auto-restart` | `on-boot` | After normal termination   | On forced close            | On Reboot                           | After package installation | After package update       |
+|----------------|-----------|----------------------------|----------------------------|-------------------------------------|----------------------------|----------------------------|
+| `FALSE`        | `FALSE`   | Not launched automatically | Not launched automatically | Not launched after reboot           | Not launched               | Not launched automatically |
+| `FALSE`        | `TRUE`    | Not launched automatically | Not launched automatically | Launched automatically after reboot | Launched                   | Launched automatically     |
+| `TRUE`         | `FALSE`   | Launched automatically     | Launched automatically     | Not launched after reboot           | Not launched               | Launched automatically     |
+| `TRUE`         | `TRUE`    | Launched automatically     | Launched automatically     | Launched automatically after reboot | Launched                   | Launched automatically     |
+
+## Related information
+  * Dependencies
```

Comment:
Although the "*" appears as a bullet point in the staging area, requesting to use a dash "-" here for consistencies sake. 
```suggestion
  - Dependencies
```